app/views/api/backend_api_configs/new.html.slim

```diff
@@ -1,4 +1,14 @@
 = content_for :sublayout_title, 'Add Backend'
 
 = semantic_form_for @backend_api_config, url: admin_service_backend_api_configs_path(@service) do |form|
-  = render 'form', form: form
+  = form.inputs do
+    = form.input :backend_api_id,
+                  as: :select,
+                  collection: current_account.backend_apis.not_used_by(@backend_api_config.service_id),
```

Comment:
```suggestion
                  collection: current_account.backend_apis.accessible.not_used_by(@backend_api_config.service_id),
```
Sorry for not noticing before :see_no_evil:

Reply:
Since we already have 3 PRs inside of one, why not a 4th? 🙂 Sure. I'll do it.